<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 68, 68);
font-size: 14pt;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 253, 255);
font-size:28pt;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Add a Class</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 232, 162);
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Class Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="className">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="classEntered">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 197, 146)</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classLink">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 232, 162);
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>CourseLink</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="meetingLink">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 232, 162);
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Meeting Link</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
